citi-onboarding/Delta-odonto
# Feature/services and our services components

## What I did
- Made the front view for Services section, including mobile and desktop versions.
## How to test
- On terminal you should type de following commands to access the server folder and run the index file:
```
cd server
node index.js
```
- Open a second terminal on client folder and run the following commands to habilite the Keystone interface
```
cd client
yarn server
```

- Go to a browser and access the adress http://localhost:3000 and you should be able to see the Services component along with several others on the landing page.

- You should be able to interact with Services component section and check if it is according with the mockup.
